WordPress/src/main/java/org/wordpress/android/ui/mediapicker/loader/DeviceListBuilder.kt

```diff
@@ -69,7 +69,7 @@ class DeviceListBuilder(
             // This item sets the threshold for the visible items in all the list
             val lastShownTimestamp = results.fold(0L) { timestamp, (_, result) ->
                 val nextTimestamp = result?.nextTimestamp
-                if (nextTimestamp != null && nextTimestamp > timestamp) {
+                if (result?.items?.isNotEmpty() == true && nextTimestamp != null && nextTimestamp > timestamp) {
```

Comment:
@planarvoid pinging you as I see you've worked on this a while back. Adding the `result?.items?.isNotEmpty() == true` effectively prevents any media from affecting the `lastShownTimestamp` when it's empty. This can happen with the new [Android 14 image partial access feature](https://developer.android.com/about/versions/14/changes/partial-photo-video-access).

I wasn't able to to reproduce on an emulator. But I can with my Samsung Galaxy Z Fold 5. Just want to run this change by you and make sure I'm not missing something.

Reply:
@irfano I need more time to investigate this further and I'll provide an update.